pyqtgraph/exporters/SVGExporter.py

```diff
@@ -358,22 +388,22 @@ def correctCoordinates(node, defs, item, options):
                 # If coords start with L instead of M, then the entire path will not be rendered.
                 # (This can happen if the first point had nan values in it--Qt will skip it on export)
                 if newCoords[0] != 'M':
-                    newCoords = 'M' + newCoords[1:]
+                    newCoords = f'M{newCoords[1:]}'
                 ch.setAttribute('d', newCoords)
             elif ch.tagName == 'text':
-                removeTransform = False
+                removeTransform = True
```

Comment:
Whoops, need to udno this change I made... I enabled it for testing purposes, should comment out again...